accelerometer/summariseEpoch.py

```diff
@@ -218,11 +219,14 @@ def check_daylight_savings_crossover(e, startTime, endTime, summary,
         for t in localTime._utc_transition_times:
             if t>startTime:
                 transition = t
+                transition = transition - pd.DateOffset(seconds = 1)
+                transition = utc_tz.localize(transition)
+                transition = localTime.normalize(transition)
```

Comment:
"normalize" the transition from UTC to local timezone

the outcome of this line, for US/eastern:
1. spring crossover: 6:59:59am -> 1:59:59am US/eastern
1. fall crossover: 5:59:59am -> 1:59:59am US/eastern